File: src/main/java/com/randomperson22/piggymodtest/entities/grandmother/EntityGrandmother.java
package com.randomperson22.piggymodtest.entities.grandmother;

import com.randomperson22.piggymodtest.entities.EntityPiggyBase;
import com.randomperson22.piggymodtest.init.ModSounds;
import net.minecraft.util.SoundEvent;
import net.minecraft.world.World;

public class EntityGrandmother extends EntityPiggyBase {
    public EntityGrandmother(World worldIn) {
        super(worldIn);
    }

 @Override
 public int getPriority() {
     return 1;
 }

    @Override
    public SoundEvent getJumpscareSound() {
        return ModSounds.ENTITY_GRANDMOTHER_JUMPSCARE;
    }
}

File: src/main/kotlin/com/randomperson22/piggymodtest/entities/grandmother/EntityGrandmother.kt
package com.randomperson22.piggymodtest.entities.grandmother

import com.randomperson22.piggymodtest.entities.EntityPiggyBase
import com.randomperson22.piggymodtest.init.ModSounds
import net.minecraft.util.SoundEvent
import net.minecraft.world.World

class EntityGrandmother(worldIn: World) : EntityPiggyBase(worldIn) {
    override val priority: Int
        get() = 1

    override val jumpscareSound: SoundEvent
        get() = ModSounds.ENTITY_GRANDMOTHER_JUMPSCARE
}